test/Microsoft.Extensions.WebSockets.Internal.Tests/Utf8ValidatorTests.cs

```diff
@@ -98,6 +101,18 @@ public void InvalidSingleFramePayloads(byte[] payload)
 
         [Theory]
 
+        [InlineData(new byte[] { 0xC0 }, new byte[] { 0xA0 })]
+        [InlineData(new byte[] { 0xC1 }, new byte[] { 0x88 })]
+        [InlineData(new byte[] { 0xF5 }, new byte[] { 0x80, 0x80, 0x80 })]
+        public void InvalidMultiFramePayloadsByFirstByte(byte[] payload, byte[] _)
```

Comment:
Just go ahead and remove the second parameter from this test. Maybe put a short comment on the test explaining why these cases are invalid.